armory/included/modules/Ingestor.py

```diff
@@ -231,9 +231,7 @@ def reclassify_domain(self, bd):
         if bd.meta.get("whois", False):
             display_new("Whois data found for {}".format(bd.domain))
             print(bd.meta["whois"])
-            res = six.input(
```

Comment:
You can probably remove the `import six` on line 13 since it appears this was the only reference to it.